src/Components/Server/src/Circuits/CircuitHost.cs

```diff
@@ -545,7 +545,7 @@ private string GetClientErrorMessage(Exception exception, string additionalInfor
             else
             {
                 return $"There was an unhandled exception on the current circuit, so this circuit will be terminated. For more details turn on " +
-                    $"detailed exceptions in '{typeof(CircuitOptions).Name}.{nameof(CircuitOptions.DetailedErrors)}'. {additionalInformation}";
+                    $"detailed exceptions by setting 'detailedErrors: true' in 'appSettings.Development.json' or set '{typeof(CircuitOptions).Name}.{nameof(CircuitOptions.DetailedErrors)}'. {additionalInformation}";
```

Comment:
Don't we usually PascalCase config values?

Reply:
no. It's json

Reply:
https://github.com/aspnet/AspNetCore/blob/a4d9faefe2ed6fbebd500413032336607ec92778/src/Hosting/Abstractions/src/WebHostDefaults.cs#L13

Reply:
But look at the appSettings.json files in our templates - they are all PascalCased.

Reply:
updated the casing to make you happy.